packages/cli-dashboard/src/components/siteReport/tabs/cookies/cookiesListing/index.tsx

```diff
@@ -71,6 +71,7 @@ const CookiesListing = ({
           tablePersistentSettingsKey={tablePersistentSettingsKey}
           selectedFrame={selectedFrameUrl}
           selectedFrameCookie={selectedFrameCookie}
+          hideExport={true}
```

Comment:
Is it necessary to pass `true` since it will be true just if we pass the attribute?
See this `showFramesSection`
https://github.com/GoogleChromeLabs/ps-analysis-tool/blob/85112cdd9f2f38195a8da876872590d120e48b5f/packages/extension/src/view/devtools/components/cookies/index.tsx#L111-L119